release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -682,6 +710,22 @@ $ operator-sdk new <operator_name> --type=helm \
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1874754[*BZ#1874754*])
 
+* In {product-title} 4.6, the resource metrics API server provides support for custom metrics. The resource metrics API server does not implement the OpenAPI specification and the following messages are sent to the `kube-apiserver` logs every 60 seconds:
++
+[source,terminal]
+----
+controller.go:114] loading OpenAPI spec for "v1beta1.metrics.k8s.io" failed with: OpenAPI spec does not exist
+controller.go:127] OpenAPI AggregationController: action for item v1beta1.metrics.k8s.io: Rate Limited Requeue.
+----
++
+In some cases, the above errors might cause the `KubeAPIErrorsHigh` alert to fire.
++
+There is no known degradation of {product-title} functionality caused by this issue.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1819053[*BZ#1819053*])
+
+* Rules API back-ends are sometimes missed if Store API stores are detected before Rules API stores. When this occurs, a store reference is created without a Rules API client and the Rules API endpoint from Thanos Querier does not return any rules.
```

Comment:
```suggestion
* Rules API back-ends are sometimes missed if Store API stores are detected before Rules API stores. When this occurs, a store reference is created without a Rules API client, and the Rules API endpoint from Thanos Querier does not return any rules.
``` 

Is there a better term than "missed?"

Reply:
@kalexand-rh I have changed this to the following:

"Rules API back-ends are sometimes not detected if Store API stores are discovered before Rules API stores."